lib/squib/args/unit_conversion.rb

```diff
@@ -13,6 +13,8 @@ def parse(arg, dpi=300)
         case arg.to_s.rstrip
         when /in$/ # ends with "in"
           arg.rstrip[0..-2].to_f * dpi
+        when /pt$/ # ends with "in"
+          arg.rstrip[0..-2].to_f * dpi / 72
```

Comment:
Done. Have plugged the constant into `save_pdf` as well as it explains the code better as well.